modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -1270,6 +1274,74 @@ else if (Objects.equals(
 			objectRelationship);
 	}
 
+	private void _bindObjectDefinitions(ObjectRelationship objectRelationship)
+		throws PortalException {
+
+		objectRelationship.setDeletionType(
+			ObjectRelationshipConstants.DELETION_TYPE_CASCADE);
+		objectRelationship.setEdge(true);
+
+		objectRelationship =
+			objectRelationshipLocalService.updateObjectRelationship(
+				objectRelationship);
+
+		_objectFieldLocalService.updateRequired(
+			objectRelationship.getObjectFieldId2(), true);
+
+		ObjectDefinition objectDefinition1 =
+			_objectDefinitionPersistence.findByPrimaryKey(
+				objectRelationship.getObjectDefinitionId1());
+
+		String objectDefinition1PreviousRESTContextPath =
+			objectDefinition1.getRESTContextPath();
```

Comment:
Can you inline since this variable is used once?

Reply:
In this case we cannot inline, because since the getRESTContextPath() is calculated based on the rootObjectDefinitionId it will not be the right one at the point it is used.